src/AcceptanceTests/SetupFixture.cs

```diff
@@ -35,17 +35,19 @@ public void OneTimeSetUp()
             var transportConfiguration = new TransportExtensions<SqsTransport>(new SettingsHolder());
             transportConfiguration = transportConfiguration.ConfigureSqsTransport(SqsQueueNamePrefix);
             var connectionConfiguration = new ConnectionConfiguration(transportConfiguration.GetSettings());
-            var sqsClient = AwsClientFactory.CreateSqsClient(connectionConfiguration);
-            var listQueuesResult = await sqsClient.ListQueuesAsync(connectionConfiguration.QueueNamePrefix).ConfigureAwait(false);
-            foreach (var queueUrl in listQueuesResult.QueueUrls)
+            using (var sqsClient = SqsTransportExtensions.CreateSQSClient())
             {
-                try
+                var listQueuesResult = await sqsClient.ListQueuesAsync(connectionConfiguration.QueueNamePrefix).ConfigureAwait(false);
+                foreach (var queueUrl in listQueuesResult.QueueUrls)
                 {
-                    await sqsClient.DeleteQueueAsync(queueUrl).ConfigureAwait(false);
-                }
-                catch (Exception ex)
-                {
-                    Console.WriteLine($"Exception when deleting queue: {ex}");
+                    try
+                    {
+                        await sqsClient.DeleteQueueAsync(queueUrl).ConfigureAwait(false);
+                    }
+                    catch (Exception ex)
+                    {
+                        Console.WriteLine($"Exception when deleting queue: {ex}");
```

Comment:
why Console.WriteLine  ?

Reply:
This was already there, I think it is done because then it is visible on TC in the build logs